torch/autograd/gradcheck.py

```diff
@@ -29,15 +29,16 @@ def make_jacobian(input, num_out):
             lambda x: x is not None, (make_jacobian(elem, num_out) for elem in input)))
         if not jacobians:
             return None
-        return type(input)(jacobians)
+        return type(input)(jacobians)  # type: ignore
     else:
         return None
 
 
-def iter_tensors(x, only_requiring_grad=False):
+def iter_tensors(x: Union[torch.Tensor, container_abcs.Iterable], only_requiring_grad: bool = False) -> Iterable[torch.Tensor]:
```

Comment:
Another occurrence of taking a generic (Iterable) without specifying the generic parameter

Reply:
Also, wouldn't Generator here be a more specific type for the return of this function?